.github/workflows/mobilecoin-dispatch-dev-delete.yaml

```diff
@@ -18,7 +18,7 @@ jobs:
     with:
       namespace: ${{ github.event.inputs.namespace }}
     secrets:
-      RANCHER_CLUSTER: ${{ secrets.RANCHER_CLUSTER }}
+      RANCHER_CLUSTER: ${{ secrets.RANCHER_CLUSTER_INFRA_DEV }}
```

Comment:
Please use INFRA_DEV_RANCHER_CLUSTER as the pattern here.  Secrets are already namespaces as DEV_<var>